system_metrics_collector/src/system_metrics_collector/utilities.cpp

```diff
@@ -96,7 +120,24 @@ double ComputeCpuActivePercentage(
   }
 
   const double active_time = measurement2.GetActiveTime() - measurement1.GetActiveTime();
-  const double total_time = ComputeCpuTotalTime(measurement1, measurement2);
+  const double total_time = (measurement2.GetIdleTime() + measurement2.GetActiveTime()) -
+    (measurement1.GetIdleTime() + measurement1.GetActiveTime());
+
+  return 100.0 * active_time / total_time;
+}
+
+double ComputePidCpuActivePercentage(
+  const ProcPidCpuData & measurement1,
+  const ProcPidCpuData & measurement2)
+{
+  if (measurement1.IsMeasurementEmpty() || measurement2.IsMeasurementEmpty()) {
+    RCUTILS_LOG_ERROR_NAMED("computePidCpuActivePercentage",
```

Comment:
The method name in the log doesn't match the definition.

Reply:
Fixed.